department-of-veterans-affairs/caseflow
# APPEALS-51447 Scheduled time showed in the left view is updated

Resolves https://jira.devops.va.gov/browse/APPEALS-51447

# Description
Did a fix for Scheduled time received from the backend is ahead from the value selected in the Hearing time.

## Acceptance Criteria

## Testing Plan
1. Login as BVASYELLOW user.
2. In the App selector click on Hearings and click on Current schedule.
3. From the Hearing page, select any Hearing with Legacy appeals.
<img width="826" alt="image" src="https://github.com/user-attachments/assets/19fe3a4c-0615-4a3a-bba3-cb1740813408">
4. In the Regional office filter select the filter for St.Petersburg
5. In the Daily docket page under the Actions view, select the Hearing time and save.
<img width="827" alt="image" src="https://github.com/user-attachments/assets/29b3ff86-acf2-4b1a-99c1-d1c6a48f433f">
6. From the left view, under Type/Time/RO column observe if the time is showing as per your selection from the Hearing time.
<img width="325" alt="image" src="https://github.com/user-attachments/assets/2b47b1f3-52a9-435f-a057-0b52333e4c3b">


1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
